components/uploadFormHook.tsx

```diff
@@ -2,22 +2,34 @@
 
 import type { FC } from 'react'
 import { type SubmitHandler, useForm } from 'react-hook-form'
+import { ImageInput } from './imageInput'
+import { ImagePreview } from './imagePreview'
 
 // フォームの各要素と型
-type FormData = {
+export type FormData = {
```

Comment:
This export property is not used anywhere now.
I think `export` should be removed.

Reply:
I removed `export`.